server/worker/api.go

```diff
@@ -81,3 +124,18 @@ func (api *Client) Post(endpoint string, payload []byte) ([]byte, error) {
 
 	return body, nil
 }
+
+func (api *Client) prepare(method string, endpoint string, payload []byte) (*http.Request, error) {
+	bearer := "Bearer " + api.Token
+
+	req, err := http.NewRequest(method, api.GrafanaUrl+endpoint, bytes.NewBuffer(payload))
+	if err != nil {
+		return nil, errors.New("Failed to create new HTTP Request")
+	}
+
+	req.Header.Add("Authorization", bearer)
+	req.Header.Add("Content-Type", "application/json")
```

Comment:
Remove, keep this as universal for any HTTP method. GET requests do not have body so `Content-Type` is not needed here.